RadioItem/RadioItem.module.less

```diff
@@ -5,11 +5,9 @@
 
 // Public
 .radioItem {
-	.icon {
-		margin: 0;
-	}
-
 	.applySkins({
+		height: @agate-radioitem-height;
```

Comment:
🤔 I think I see this variation a lot...
The default size of RadioItem of silicon seems "small" right?
I think I guess just pass `size="small"` to the Item when silicon skin is applied instead of overriding the size alone.

Reply:
Sounds good to me. Thanks for the tip